defaults/main/main.yml

```diff
@@ -11,4 +11,118 @@ mailcow__timezone: Europe/Berlin
 # WARNING: Only lumen is loaded locally. Enabling any other theme, will download external sources.
 mailcow__theme: lumen
 
-mailcow__install_updates: yes
\ No newline at end of file
+mailcow__install_updates: yes
+
+# ------------------------------
```

Comment:
Why did you put the mailcow settings into the main.yml there is a mailcow.yml for these kind of settings.